### PR TITLE
Chore(setup Node Server and Sequelize)

### LINKED ISSUE #3: Initialize server and set up database connetions
**What will this issue resolve?**
- Initialize the node server and setup the database connection for development

**Describe the tasks to implement**
- set up the node server in the node entry file (server.js)
- set up database connection config using sequelize
- set up env file with database connection variables

### PR DESCRIPTION
**Pull request by bright258**
Set server to listen to a port, and connected to a PostgreSQL database using Sequelize

### CODE REVIEW

**Comment by bright258** [comment]
To test:

1. Set up a PostgreSQL database either locally or online. 

2. Get your database credentials such as database name, database username, database password and database host. 

3. Create a .env file and input those credentials there using the fields DATABASE_NAME, DATABASE_USERNAME,   DATABASE_PASSWORD, DATABASE_HOST.

File: src/server.js [hunk 1]
```diff
@@ -1 +1,28 @@
...
+
+const { sequelize } = require('../database/db.config');
+
+const app = express();
+
+require('dotenv').config();
+
+const port = process.env.PORT;
+
+// database connection using configuration in db.config.js
```
Comment (kelue) [inline src/server.js]: Database connection checks should be done in the config file too. The purpose of the config file is to abstract everything about the database from the main entry file

**Review by kelue — CHANGES_REQUESTED** [review]
Move the connection check function to the db config file, same with the function call, once you import the sequelize export to the entry file, the server will still run the check as usual

File: src/server.js (hunk 1 above)
Comment (kelue) [reply src/server.js]: also port should have default port in case port is not set(best practice)
```
const port = process.env.PORT || 8080;
```

Comment (bright258) [reply src/server.js]: Noted

**Comment by bright258** [comment]
How should I add the Database.md file for describing how to test the database connection?

**Comment by kelue** [comment]
> How should I add the Database.md file for describing how to test the database connection?

Just create a DATABASE.md file or you know what, forget about the md file, we will just add all that info in the migrations file

**Comment by bright258** [comment]
I have made the requested changes. Please check them out

**Comment by bright258** [comment]
I have also created the .env.example file and the database.md file

**Review by kelue — APPROVED** [review]

**Pull request merged**